src/vs/workbench/contrib/authentication/browser/actions/manageAccountsAction.ts

```diff
@@ -131,6 +132,13 @@ class ManageAccountsActionImpl {
 			}
 		}));
 
+		store.add(quickPick.onDidTriggerButton((button) => {
+			if (button === this.quickInputService.backButton) {
+				quickPick.hide();
```

Comment:
this isn't needed since the run call will show a quick pick that will replace the one currently shown.

it'll also prevent a flicker if there's any delay between hide() and run()